File: JavaScript/M00_JavaScriptProgrammingBasics/L02_ConditionalStatements/Exercises/P01_SumSeconds.js
function sumSeconds(a, b, c) {
    let timeFirst = Number(a);
    let timeSecond = Number(b);
    let timeThird = Number(c);

    let totalTime = timeFirst + timeSecond + timeThird;

    let minutes = Math.floor(totalTime / 60);
    let seconds = totalTime % 60;

    if (seconds < 10) {
        console.log(`${minutes}:0${seconds}`);
    } else {
        console.log(`${minutes}:${seconds}`);
    }
}

File: Python/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P01_SumSeconds.py
time_first = int(input())
time_second = int(input())
time_third = int(input())

time_total = time_first + time_second + time_third

time_minute = 0
time_seconds = 0

if time_total / 60 >= 0:
    time_minute = time_total // 60
    time_second = time_total % 60

if time_second < 10:
    time_second = "0" + str(time_second)

print(str(time_minute) + ":" + str(time_second))